lib/tasks/package/npm.js

```diff
@@ -20,6 +20,13 @@ const buildNpmPackageJson = () => {
   npmPackageJson['license'] = packageJson.license
   npmPackageJson['bugs'] = packageJson.bugs
   npmPackageJson['homepage'] = packageJson.homepage
+  // THIS IS A LATE NIGHT HACK, MAYBE JUST USE THE ORIGINAL PACKAGE.JSON SOMEHOW
+  npmPackageJson['dependencies'] = {
+    "@frctl/fractal": "^1.0.8",
+    "@frctl/nunjucks": "^1.0.1",
+    "run-sequence": "^1.2.2"
+  }
+
```

Comment:
this seems messy, how do we get dependencies in from the package.json in a clean way?